src/richie/apps/courses/templates/courses/cms/course_detail.html

```diff
@@ -172,6 +172,28 @@ <h1 class="subheader__title" property="name">{% render_model current_page "title
                     {% block contact %}
                         <a href="#" class="subheader__cta">{% trans "Contact us" %}</a>
                     {% endblock contact %}
+
+                    {% block run_open_single %}
+                        {% with runs_dict=course.course_runs_dict %}
+                            {% with open_visible_runs=runs_dict.0|add:runs_dict.1|add:runs_dict.2|visible_on_course_page:request.toolbar.edit_mode_active %}
+                                {% if open_visible_runs|length <= 1 %}
+                                    <div class="course-detail__row course-detail__runs course-detail__runs--open">
+                                        {% for run in open_visible_runs|slice:":1" %}
+                                            {% include "courses/cms/fragment_course_run.html" %}
+                                        {% endfor %}
+                                        {% if open_visible_runs|length == 0 %}
+                                            <div class="course-detail__empty">{% trans "No open course runs" %}</div>
+                                        {% endif %}
+                                    </div>
+                                {% endif %}
+                                <div class="course-detail__row course-detail__runs">
+                                    {% if open_visible_runs|length > 1 %}
+                                        <a href="javascript:void(0);" onclick="document.getElementById('courseDetailsRunsOpen').scrollIntoView({behavior: 'smooth', block: 'start'});" onclick="">{{ open_visible_runs|length }} {% trans "course runs are currently open for this course" %}</a>
+                                    {% endif %}
+                                </div>
```

Comment:
What do you think to reorganize thinks like that ? First for readability, second to prevent an empty div when `open_visible_runs` length is `<= 1`;
```suggestion
                                {% if open_visible_runs|length == 0 %}
                                    <div class="course-detail__empty">{% trans "No open course runs" %}</div>
                                {% elif open_visible_runs|length == 1 %}
                                    <div class="course-detail__row course-detail__runs course-detail__runs--open">
                                        {% for run in open_visible_runs|slice:":1" %}
                                            {% include "courses/cms/fragment_course_run.html" %}
                                        {% endfor %}
                                    </div>
                                {% else %}
                                    <div class="course-detail__row course-detail__runs">
                                        <a href="javascript:void(0);" onclick="document.getElementById('courseDetailsRunsOpen').scrollIntoView({behavior: 'smooth', block: 'start'});" onclick="">{{ open_visible_runs|length }} {% trans "course runs are currently open for this course" %}</a>
                                    </div>
                                {% endif %}
```

Reply:
Done!